pkg/products/rhssouser/prometheusRules.go

```diff
@@ -72,6 +73,77 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+
+			//SLO-availability-alerts:
+			//https://sre.google/workbook/alerting-on-slos/
+			//https://promtools.dev/alerts/errors
+			{
+				AlertName: "user-sso-slo-availability-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "user-sso-slo-availability.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: fmt.Sprintf("%sUserSsoAvailability5mto1hErrorBudgetBurn", strings.ToUpper(installationName)),
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlSloUserSsoAvailabilityAlert,
+							"message": "High 5m and 1h error budget burn for SSO User",
+						},
+						Expr: intstr.FromString(fmt.Sprintf(`
+							sum( sum(rate(haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace='%s', code="5xx"}[5m]))
```

Comment:
same applies here I don't think we need the regex and 99% is ok as a starting point but will soon want to up to 99.95 once we see not making too much noise. We will also want to move them to critical that point

Reply:
I wanted to ask - maybe we could progress to approval/merge with current Alerts task and this PR?
Current development was verified both for RHOAM and RHMI.
We could open new tasks for improvements including testing automation.